CONTRIBUTING_ADDITONAL_TIPS.md

```diff
@@ -0,0 +1,104 @@
+# Tips for Contributing to Docusaurus
+
+This file expands on the Code Convenions introduced in the Code Convenions' section in CONTRIBUTING.md
+
+"Code Conventions
+Most important: Look around. Match the style you see used in the rest of the project. This includes formatting, naming files, naming things in code, naming things in documentation, etc."
+
+# Code Contribution Helpful Tips: 
+
+If you are using Visual Studio Code you can wrap lines by clicking alt + z.
+
+# To make your code ‘Attractive’ try using the following conventions:
+
+ ## Naming files
+
+ File are typically named using dash seperation
+ 
+   Examples: 
+   docusaurus/admin/local-third-party-project-testing.md
+   docusaurus/__tests__/validate-package-json.test.ts
+   docusaurus/website/docs/browser-support.md
+   docusaurus/website/docs/migration/migration-translated-sites.md
+
+ ## Naming things in code
+
+  camelCase naming is typically in config.js files
+  
+  Ex 1: docusaurus/examples/classic/docusaurus.config.js 
+
+    export type ActiveDocContext = {
+    activeVersion?: GlobalVersion;
+    activeDoc?: GlobalDoc;
+    alternateDocVersions: {[versionName: string]: GlobalDoc};
+    };
+
+  Dash separated naming is typically used in .json files
```

Comment:
We use camelCase. Period. No other casing is allowed in code. Dash naming for comamnds is universal; dash naming for packages is because they are literally called that.

It's not something we have to talk about in CONTRIBUTING, rather it should be enforced by a linter.